test/functional/rpc_psbt.py

```diff
@@ -620,7 +620,7 @@ def test_psbt_input_keys(psbt_input, keys):
         wallet = self.nodes[1].get_wallet_rpc("extfund")
 
         # Make a weird but signable script. sh(pkh()) descriptor accomplishes this
```

Comment:
9a8cf704e51961894f8d687c58195dbb31cf53f5: comment should be updated to say sh(wsh(pkh())) since you've changed the descriptor type

Reply:
Done